tests/storage/cdi_upload/test_upload.py

```
@@ -295,14 +295,15 @@ def _upload_image(dv_name, namespace, storage_class, local_name, client):
     indirect=True,
 )
 def test_successful_concurrent_uploads(
+    admin_client,
     unprivileged_client,
     upload_file_path,
     namespace,
     storage_class_matrix__module__,
 ):
     dvs_processes = []
     storage_class = [*storage_class_matrix__module__][0]
```

Ruff (0.16.1)

[warning] Prefer `next(iter(storage_class_matrix__module__))` over single element slice

Replace with `next(iter(storage_class_matrix__module__))`

(RUF015)